pkgs/development/pharo/vm/build-vm-legacy.nix

```diff
@@ -0,0 +1,72 @@
+{ stdenv, fetchurl, cmake, bash, unzip, glibc, openssl, gcc, mesa, freetype, xorg, alsaLib, cairo, ... }:
+
+{ name, src, ... }:
+
+stdenv.mkDerivation rec {
```

Comment:
Is there a use case for keeping the legacy around? If someone really needs it it's still there in previous revisions of nixpkgs.